app/ts/components/simulationExplaining/SimulationSummary.tsx

```diff
@@ -665,12 +665,11 @@ export function SimulationSummary(param: SimulationSummaryParams) {
 	if (ownAddresses === undefined || notOwnAddresses === undefined) throw new Error('addresses were undefined')
 
 	const icon = useComputed(() => {
-		for (const block of param.simulationAndVisualisationResults.visualizedSimulationState.visualizedBlocks) {
-			const failure = block.simulatedAndVisualizedTransactions.find((transaction) => transaction.statusCode !== 'success')
-			if (failure) return '../img/error-icon.svg'
-			const quarantine = block.simulatedAndVisualizedTransactions.find((transaction) => transaction.quarantine)
-			if (quarantine) return '../img/warning-sign.svg'
-		}
+		const transactions = param.simulationAndVisualisationResults.visualizedSimulationState.visualizedBlocks.flatMap((block) => block.simulatedAndVisualizedTransactions)
+		const failure = transactions.find((transaction) => transaction.statusCode !== 'success')
+		if (failure) return '../img/error-icon.svg'
```

Comment:
Consider alternatives (same for below):
```suggestion
		if (transactions.some((transaction) => transaction.statusCode !== 'success') '../img/error-icon.svg'
```
or
```suggestion
		const isError = (transaction: typeof transaction[number]) => transaction.statusCode !== 'success'
		if (transactions.some(isError) '../img/error-icon.svg'
```